packages/fhir-converter/src/templates/cda/Resources/Immunization.hbs

```diff
@@ -29,13 +29,18 @@
                     "lotNumber":"{{this.manufacturedMaterial.lotNumberText._}}",
                 {{/if}}
             {{/each}}
+            
+            {{#with this.manufacturedProduct.manufacturerOrganization}}
+                {{!-- manufacturer should be a reference to an Organization --}}
+                "manufacturer": {
```

Comment:
Note that this should be a Reference to an Organization (manufacturer). I haven't been able to make that work.. But the MR generation with this approach allows us to display the manufacturer's name, at least